Makefile

```
@@ -3,24 +3,22 @@ include .env
 .PHONY: dev env/setup env/teardown codebase codebase/fix
 
 dev:
-	make install-dependencies
-	make env/setup
-	./bin/dev
+	bundle exec make install-dependencies
+	bundle exec make env/setup
```

```suggestion
	make install-dependencies
	make env/setup
```

In a similar way, `make` command in general shouldn't need it